### ISSUE TITLE
Firmware update causes FirmwareError

### ISSUE THREAD

**Issue by prusnak** [trezor/trezor-firmware#122]
Firmware 2.1.1 (unreleased - 2883f03583350249ef42a139141a9ee962e82a6a from branch `release/2019-05`).

When I use `trezorctl get-public-node` or `trezorctl get-address` on newly initialized device it works. However, if I update from 2.1.0 and use any of the above I get `FirmwareError` exception.

**Comment by prusnak** [comment]
Exception:

```
1556658418802104 trezor.wire ERROR exception:
Traceback (most recent call last):
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/wire/__init__.py", line 141, in session_handler
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/wire/__init__.py", line 138, in session_handler
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/wire/__init__.py", line 178, in protobuf_workflow
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/wire/__init__.py", line 165, in protobuf_workflow
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/wire/__init__.py", line 185, in keychain_workflow
  File "/home/stick/work/trezor/trezor-firmware/core/src/apps/common/seed.py", line 67, in get_keychain
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/ui/__init__.py", line 146, in inner
  File "/home/stick/work/trezor/trezor-firmware/core/src/trezor/ui/__init__.py", line 144, in inner
  File "/home/stick/work/trezor/trezor-firmware/core/src/apps/common/seed.py", line 86, in _compute_seed
  File "/home/stick/work/trezor/trezor-firmware/core/src/apps/common/mnemonic.py", line 21, in get_seed
NameError: local variable referenced before assignment
```

**Comment by prusnak** [comment]
Variable `module` is not set on line 21 if the storage is migrated from fw version 2.1.0 or older

**Comment by prusnak** [comment]
Fixed via 1812be5d49a51d7e6cbc3d251b4794626107fc7a
